src/app/home/page.tsx

```diff
@@ -0,0 +1,169 @@
+  const views = 123;
+  const comments = 456;
+  const likes = 789;
+
+  return (
+    <Box
+      sx={{
+        width: "100%",
+        maxWidth: "500px",
+        backgroundColor: "#FFFFFF",
+        padding: "32px",
+        borderRadius: "16px",
+        border: "1px solid #D6D6E7",
+        boxShadow: "0px 4px 12px rgba(0, 0, 0, 0.1)",
+        display: "flex",
+        flexDirection: "column",
+        gap: "24px",
+        alignItems: "center",
+      }}
+    >
+      <Box
+        sx={{
+          width: "100%",
+          display: "flex",
+          justifyContent: "space-between",
+          alignItems: "center",
+        }}
+      >
+        <Typography
+          variant="h6"
+          sx={{ fontWeight: 600, color: "#333", fontFamily: "Outfit, sans-serif" }}
+        >
+          {name}
+        </Typography>
+        <Typography
+          variant="body2"
+          sx={{
+            color: "#6C6C80",
+            fontSize: "16px",
+            lineHeight: "20.16px",
+            fontFamily: "Outfit, sans-serif",
+          }}
+        >
+          {date}
+        </Typography>
+      </Box>
+
+      <Box
+        sx={{
+          width: "400px",
+          height: "400px",
+          backgroundColor: "#F0F0F5",
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          borderRadius: "8px",
+        }}
+      >
+        <Typography
+          sx={{
+            color: "#6C6C80",
+            fontSize: "16px",
+            fontWeight: 400,
+            fontFamily: "Outfit, sans-serif",
+          }}
+        >
+          Name or Logo or Demo
+        </Typography>
+      </Box>
+
+      <Box
+        sx={{
+          width: "100%",
+          display: "flex",
+          justifyContent: "space-around",
+          alignItems: "center",
+        }}
+      >
+        <Box sx={{ textAlign: "center", display: "flex", alignItems: "center", gap: "8px" }}>
+          <VisibilityOutlinedIcon sx={{ color: "#000000", fontSize: "36px" }} />
+          <Typography sx={{ fontSize: "14px", color: "#333", fontFamily: "Outfit, sans-serif" }}>
+            {views}
+          </Typography>
+        </Box>
+        <Box sx={{ width: "1px", backgroundColor: "#D6D6E7", height: "48px" }} />
+
+        <Box sx={{ textAlign: "center", display: "flex", alignItems: "center", gap: "8px" }}>
+          <ModeCommentOutlinedIcon sx={{ color: "#000000", fontSize: "36px" }} />
+          <Typography sx={{ fontSize: "14px", color: "#333", fontFamily: "Outfit, sans-serif" }}>
+            {comments}
+          </Typography>
+        </Box>
+        <Box sx={{ width: "1px", backgroundColor: "#D6D6E7", height: "48px" }} />
+
+        <Box sx={{ textAlign: "center", display: "flex", alignItems: "center", gap: "8px" }}>
+          <FavoriteBorderOutlinedIcon sx={{ color: "#000000", fontSize: "36px" }} />
+          <Typography sx={{ fontSize: "14px", color: "#333", fontFamily: "Outfit, sans-serif" }}>
```

Comment:
unless this is specific to this boxes I assume this should be a property set in the global.css or theme provider to ensure consistency across the application.

Reply:
Created global variables for that